docs/_Research-Data-Management/13-data-qc.md

```diff
@@ -1,196 +1,387 @@
+
+
+### 🚩 Inconsistent Replicates (Post-Mapping)
+
+within replicate correlation is weak (r2<0.5). 
+
+Possible reasons are: e.g. contamination with HVG like rRNAs. 
+
+💡 if only a small sub-population is confounding -> remove  
+
+replicates do not cluster. 
+
+Possible reasons are: e.g. biol. and tech. replicates are mixed up. 
+
+💡 END-RESTART  
+
+replicates do not cluster. 
+
+Possible reasons are: negative control study. 
+
+
+
+
+### 🚩 Correlation between biological conditions (Post-Mapping)
+
+two experiment of 2 conditions (ctrl, KO) cluster in the wrong area. 
+
+Possible reasons are: human error, e.g. misslabeling of tubes. 
+
+💡 check design matrix, documentation, ...  
+
+no clear separation between conditions (ctrl, KO) . 
+
+Possible reasons are: e.g. biol. and tech. replicates are mixed up. 
+
+💡 END-RESTART  
+
+
+### 🚩 Dispersion Plot Anomalies (DEA)
+
+dispersion-plot: gene estimation does not follow red fit. 
+
+Possible reasons are: model does not represent data well. 
+
+💡 Deseq is not applicable  
+
+dispersion-plot: high fit dispersion for high mean count. 
+
+Possible reasons are: low number of replication + high variability. 
+
+💡 careful with reported DEGs  
+
+
+### 🚩 Predominantly Differentially Expressed Genes (DEA)
+
+almost all genes are DE (>90%). 
+
+Possible reasons are: e.g. biol. and tech. replicates are mixed up. 
+
+💡 END-RESTART  
+
+
+### 🚩 Sparse Differentially Expressed Genes (DEA)
+
+almost no gene is DE (<5%). 
+
+Possible reasons are: negative control study. 
+
+
+
+
+### 🚩 Wrong visualization metric (Visualize)
+
+using CPM to visualize DEGs. 
+
+Possible reasons are: mixed up within- and between-sample normalization. 
+
+💡 use correct normalization  
+
+use raw Ratios to visualize DEGs effect size. 
+
+Possible reasons are: humans are bad with ratios (0.01 = almost 0 and 100 is just large but not the largest bar ever). 
+
+💡 use any log transformation (e.g. log10: 0.01 => -2, 100 => +2)  
+
+
+# single cell
+
+
+### 🚩 Problem with per cell histogram (quality check)
+
+peak at left/right side in gene or reads per cell histogram or log10-cummulative-number of reads per cell id. 
+
+Possible reasons are: left=cell fragments, right=multiplets present. 
+
+💡 remove with cut-offs  
+
+
+### 🚩 inflated UMI counts (quality check)
+
+inflated UMI counts. 
+
+Possible reasons are: using raw UMI counts. 
+
+💡 use DBEC/RSEC UMI counts  
+
+
+### 🚩 Suboptimal embedding (dimension reduction)
+
+poor PCA/UMAP/tSNE embedding. 
+
+Possible reasons are: using e.g. only one assay of count data for embedding. 
+
+💡 for multimodal data use a WNN approach (combining both assays)  
+
+poor PCA/UMAP/tSNE embedding. 
+
+Possible reasons are: use raw data for tSNE/UMAP. 
+
+💡 use a significant portion of PC from the PCA as input for tSNE/UMAP  
+
+
+### 🚩 Wrong embeddings (dimension reduction)
+
+"The first two principle components were used to perform a tSNE" (https://www.science.org/doi/full/10.1126/science.aag3009). 
```

Comment:
```suggestion
"The first two principle components were used to perform a tSNE" (https://doi.org/10.1126/science.aag3009). 
```

This is the doi and the URL is already in the ignore file for the bot check since 7bb439c400fca73c42ac0a3f86c17272dca5472b